src/test/java/nextstep/subway/line/LineTestFixture.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.subway.line;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.springframework.http.MediaType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LineTestFixture {
+    public static ExtractableResponse<Response> 지하철_노선_등록되어_있음(String name, String color){
```

Comment:
```suggestion
    public static ExtractableResponse<Response> 지하철_노선_등록되어_있음(String name, String color) {
```

IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.

Reply:
Auto Formatting 기능을 사용을 하고는 있었는데
전체 소스에 적용이 가능한지 몰라서 파일 몇개가 빠졌었네요 ^^;
이제 한번에 적용하는 법을 알게됐습니다 !